src/monster.php

```diff
@@ -1056,16 +1058,16 @@
 			echo "</tr></table>";
 			echo "</div>";
 		} elseif (($bixo->type == 98 || $bixo->type == 99) && $bixo->hp > 0 || $player->hp <= 0 || $morreu == 5) {
-			echo showAlert("<a href=\"javascript:void(0)\" onclick=\"javascript:LoadPage('heal.php', 'swap')\">Clique aqui</a> para recuperar toda sua vida por <b>" . number_format($cost) . '</b> de ouro. | <a href="monster.php">Voltar</a>', "white", "left");
+			echo showAlert("<a href=\"#\" onclick=\"javascript:LoadPage('heal.php', 'swap')\">Clique aqui</a> para recuperar toda sua vida por <b>" . number_format($cost) . '</b> de ouro. | <a href="monster.php">Voltar</a>', "white", "left");
 		} else {
 			echo '<table width="100%" height="43px" border="0px"><tr><td width="85%" bgcolor="#E1CBA4">';
-			echo "<a href=\"javascript:void(0)\" onclick=\"javascript:LoadPage('swap_type.php?type=97', 'swap')\"><img src=\"static/images/magias/hit.png\" style=\"border: 0px; padding-top: 3px; padding-left: 5px; z-index: 3;\" border=\"0\" /></a>";
+			echo "<a href=\"#\" onclick=\"javascript:LoadPage('swap_type.php?type=97', 'swap')\"><img src=\"static/images/magias/hit.png\" style=\"border: 0px; padding-top: 3px; padding-left: 5px; z-index: 3;\" border=\"0\" /></a>";
```

Comment:
By changing the href from javascript:void(0) to # the page will be scrolled to the top, whereas not changing it would make the page not scroll, is it the intended behavior?